src/UIKit/UIAccessibility.cs

```diff
@@ -300,18 +326,24 @@ static unsafe void TrampolineRequestGuidedAccessSession (IntPtr block, bool enab
 			}
 		}
 
-#if !NET
+#if NET
```

Comment:
This is re-introducing a member to .NET that we don't want (the `#if !NET` surrounded the entire property before the change, but after the change only the availability attributes).

Reply:
Good catch.